README.md

```diff
@@ -78,3 +78,11 @@ The following example configures Fast-RTPS to publish synchronously, and to have
         ```bash
         FASTRTPS_DEFAULT_PROFILES_FILE=<path_to_xml_file> RMW_FASTRTPS_USE_QOS_FROM_XML=1 RMW_IMPLEMENTATION=rmw_fastrtps_cpp ros2 run demo_nodes_cpp listener
         ```
+
+## Quality Declaration files
+
+Quality Declarations for the external dependencies of these packages can be found in:
```

Comment:
We've been stating the quality declaration in the README as well. See https://github.com/ros2/rmw/blob/master/rmw/README.md for an example